lib/datapacktypes/apexclass.js

```diff
@@ -0,0 +1,11 @@
+var ApexClass = module.exports = function(vlocity) {
+    this.vlocity = vlocity;
+};
+
+ApexClass.prototype.onAutoExport = function(refObject) {
+    if(refObject.jobInfo.includeSalesforceMetadata && !refObject.currentData.remoteClass.includes("%vlocity_namespace%")){
+        return 'classes/'+ refObject.currentData.remoteClass + '.cls';
+    } else{
```

Comment:
This is not actually necessary. If you have no return it will return null automatically.